content/en/docs/reference/glossary/Quantity.md

```diff
@@ -0,0 +1,24 @@
+---
+title: Quantity
+id: Qty
+date: 2023-01-24
+full_link: https://kubernetes.io/docs/tasks/configure-pod-container/assign-memory-resource/
+short_description: >
+ Quantity contains GB and GiB unit. 
+GiB: Gibibyte 
+GB: Gigabyte.
+
+aka: 
+tags:
+- user-type
+---
+
+
+
+
+
+
+
+
+
+
```

Comment:
You need to write the text for the glossary entry @Sajiyah-Salat

Reply:
I will try to add.